base/markdown/render/rich.jl

```diff
@@ -3,7 +3,7 @@ function tohtml(io::IO, m::MIME"text/html", x)
 end
 
 function tohtml(io::IO, m::MIME"text/plain", x)
-    writemime(io, m, x)
+    printesc(io, sprint(writemime, m, x))
```

Comment:
@one-more-minute Another example where this kind of thing is useful is if we had newline characters in the MD (which the spec suggests), in html you want to print them as new lines BUT in the terminal as spaces (as html is rendered). Somehow:

    remove_newlines(io, terminalinline(..., x))